### ISSUE TITLE
SD card is not detected

### ISSUE THREAD

**Issue by blammit** [victronenergy/gui-v2#1275]
When an SDCard is installed and being used for VRM logging, not correctly detected. See screen shot from the v1 and v2 GUI.

![image](https://github.com/victronenergy/gui-v2/assets/1260099/292a9d53-2a14-42bb-8072-e9f317e9a890)
![image](https://github.com/victronenergy/gui-v2/assets/1260099/0da5d508-fa37-4c5b-86ee-23f6e6adab50)

**Comment by ReinvdZee** [comment]
In gui-v1, the gui's backend connects to some dbus signals: https://github.com/victronenergy/gui/blob/master/platform/storage_events_dbus.cpp

Unmounting is done by requesting `vrmlogger` to do so: https://github.com/victronenergy/gui/blob/master/qml/MbMountState.qml

`VrmLogger` already also handles mount events; a storage medium will be used for logging when inserted; even without gui-v1 running. 

I tested this on an Ekrano GX with gui-v2 running locally and it seems to work properly (both local and wasm display correct values):

![image](https://github.com/user-attachments/assets/d2ede4d2-8888-4fcd-b427-579df6ce8949)


@blammit is this still an issue?

**Comment by blammit** [comment]
@nmbath could you please re-test with the latest version?

**Comment by blammit** [comment]
Thank you, will close this one then.

**Comment by nmbath** [comment]
@blammit, I think this needs re-opening

Looking more closely at the screen shot I provided it is showing that Storage is internal and not external. I have posted on slack that there are issues.  https://victrondevelopment.slack.com/archives/C04RYU0UMDE/p1724420721369989

Summary:
GUI when accessing remotely does not display the same as when viewing on Touch 50.

Touch 50
![image](https://github.com/user-attachments/assets/dfec23f6-36cb-4d88-979f-f3f2f5b23ce8)

Remote
<img width="1189" alt="image" src="https://github.com/user-attachments/assets/47b07674-860b-4bc9-94c2-2ad721b6e810">

Further and not sure this is a GUI thing!  If the SDCard is inserted it is found, following a reboot it is not detected.
